index.html

```diff
@@ -0,0 +1,1783 @@
+                            <div>
+                                <h6 class="text-danger">SUBSCRIBE</h6>
+                            </div>
+                            <div class="bg-secondary container p-5 ">
+                                <div class="row m-auto">
+                                    <div class="col-3">
+                                        <img src="https://d.newsweek.com/en/full/2102837/cover-september-02-2022.jpg?w=170&q=95&f=5415e06ea39bcfa1b7450fbe86525e14"
+                                            alt="">
+                                    
+                                    </div>
+                                    <div class="col-6">
+                                        <img src="https://d.newsweek.com/en/full/2107387/cover-september-09-2022.jpg?w=170&q=95&f=6bc2e4656b8897eeb9814479288a12e7"
+                                            alt="">
+                                    </div>
+                                    <div class="col col-sm-12 text-center">
+                                        <h4>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+                                        <p>All Access + Weekly Delivery. <br>
+                                        Choose A Membership That's Perfect for You!</p>
+                                        <button type="button" class="btn btn-danger fw-bold">SUBSCRIBE</button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-lg-4">
+                            <div>
+                                <h6 class="text-danger">NEWSLETTER</h6>
+                            </div>
+                            <div class="container">
+                                <div class="row">
+                                    <div class="col-md-5 col-sm-12">
+                                        <img  src="https://d.newsweek.com/en/full/1541302/free-newsletters.jpg?w=397&h=265&l=77&t=48&q=90&f=f71406d217eb2da14441271ff0a7b831"
+                                            alt="">
+                                    </div>
+                                    <div class="col-md-7 col-sm-12 text-center bg-light  ">
+                                        <h4>Best of Newsweek via email</h4>
+                                        <p>Daily news headlines & detailed briefings enjoyed by half a million readers.</p>
+                                        <div class="d-sm-flex justify-content-center">
+                                            <input type="email" name="email" placeholder="Email address" class="">
+                                            <button type="button" class="btn btn-danger fw-bold">FREE SIGN UP</button>
+                                        </div>
+                                    </div>
+                                </div>
```

Comment:
- [ ] In the email subscribe input area at the footer, please fix the way the email input looks, it's not properly aligned like the original webpage.

| Original  | Yours |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/33323077/190873887-356aa455-267f-459a-b9d0-1237a644e999.png" width="300px"/>  | <img src="https://user-images.githubusercontent.com/33323077/190873949-e0d031b9-5648-4fd2-a72b-75ef6f3fed9a.png" width="300px"/> |